jupyterlab_search_replace/tests/test_handlers.py

```diff
@@ -2,10 +2,8 @@
 
 
 async def test_search_get(test_content, jp_fetch):
-    # When
-    response = await jp_fetch("search-regex", params={"regex": "strange"}, method="GET")
+    response = await jp_fetch("search", params={"query": "strange"}, method="GET")
 
-    # Then
     assert response.code == 200
     payload = json.loads(response.body)
```

Comment:
Could you add a test validating the payload with the JSON schema? You can use [jsonschema](https://github.com/Julian/jsonschema) for that.

Reply:
Done